pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -56,11 +56,14 @@ rec {
 
       # This used to be in-nixpkgs, but now is in the bundle
       # because I can't be bothered to make it partially static
-      cp ${curl}/bin/curl $out/bin
-      cp -d ${curl}/lib/libcurl*.dylib $out/lib
-      cp -d ${libssh2}/lib/libssh*.dylib $out/lib
+      cp ${curl-light}/bin/curl $out/bin
+      cp -d ${curl-light}/lib/libcurl*.dylib $out/lib
       cp -d ${openssl}/lib/*.dylib $out/lib
 
+      # Add ca-certs for curl verification
```

Comment:
Look at the next PR.

On Sun, May 31, 2015, 00:35 Vladimír Čunát <notifications@github.com> wrote:

> In pkgs/stdenv/darwin/make-bootstrap-tools.nix
> <https://github.com/NixOS/nixpkgs/pull/8081#discussion_r31387744>:
>
> >        cp -d ${openssl}/lib/*.dylib $out/lib
> >
> > +      # Add ca-certs for curl verification
>
> I thought bootstrap only uses fetchurl, which doesn't use certificates
> AFAIK (--insecure).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/8081/files#r31387744>.
>